### PR TITLE
Update dom element when set prompt

### PR DESCRIPTION
**Pull request by coderaiser**
When method ` SetPromptLabel` called current prompt do not updates immediately in console, it only works on next output of data.
Added ability to update prompt label, on function call to make `jq-console` more interactive and remove this delay.

### CODE REVIEW

**Comment by amasad** [comment]
I just noticed this PR. Can we have this as a separate method? Say `UpdatePromptLabel`

**Comment by coderaiser** [comment]
Done

File: src/jqconsole.coffee [hunk 1]
```diff
@@ -439,10 +439,18 @@ class JQConsole
   #   @arg continue_label: The new continuation label for the next prompt. Optional.
   SetPromptLabel: (main_label, continue_label) ->
     @prompt_label_main = main_label
+    prompt_selector = '>span+span>span:first-child'
+    $('.' + CLASS_PROMPT + prompt_selector).text(main_label);
```
Comment (amasad) [inline src/jqconsole.coffee]: I think you forgot this here

File: src/jqconsole.coffee [hunk 2]
```diff
@@ -439,10 +439,18 @@ class JQConsole
...
     if continue_label?
       @prompt_label_continue = continue_label
     return undefined
 
+  # Update the main prompt label.
+  UpdatePromptLabel: ->
+    prompt_selector = '>span+span>span:first-child'
+    full_selector = '.' + CLASS_PROMPT + prompt_selector
+    $(full_selector).text(@prompt_label_main)
```
Comment (amasad) [inline src/jqconsole.coffee]: Let's scope this to `@$console` otherwise we can unexpectedly change other console on the page.

**Comment by coderaiser** [comment]
Done

**Comment by coderaiser** [comment]
Thank you for adding this changes. Could you tell me please when would be new release?

**Comment by amasad** [comment]
I wanted to merge the other pull request and do a release, I'll do it tomorrow

**Pull request merged**